api/nedluzimstatu/_shared.ts

```diff
@@ -0,0 +1,34 @@
+import * as http from "http";
+import * as express from "express";
+
+const corsAllowedOrigins = [
+  "https://www.nedluzimstatu.cz",
+  "https://nedluzimstatu.ceskodigital.net",
+];
+const corsAllowedHeaders = "Origin, X-Requested-With, Content-Type, Accept";
+const httpCacheControl = "public, max-age=3600";
+
+export function addCORSHeaders(
+  proxyResponse: http.IncomingMessage,
+  request: express.Request,
+  _: express.Response
+) {
+  let origin = request.headers.origin;
+  let theOrigin =
+    corsAllowedOrigins.indexOf(origin) >= 0 ? origin : corsAllowedOrigins[0];
+  proxyResponse.headers["Access-Control-Allow-Origin"] = theOrigin;
+  proxyResponse.headers["Access-Control-Allow-Headers"] = corsAllowedHeaders;
+  proxyResponse.headers["Cache-Control"] = httpCacheControl;
+}
+
+export function runMiddleware(req: any, res: any, fn: any) {
```

Comment:
Tady netuším správné typy, `any` je tam čistě jako vztyčený prostředníček překladači. Víte někdo typy?

Reply:
Shodli jsme se na tom, že na to kašlem.